app/views/AssociatedCompaniesView.scala.html

```diff
@@ -61,7 +64,7 @@
         @govukRadios(
             RadiosViewModel(
                 field  = form("associatedCompanies"),
-                legend = LegendViewModel(""),
+                legend = LegendViewModel(messages(s"$heading")).withCssClass("govuk-visually-hidden"),
```

Comment:
same comment, we don't need to wrap it in `messages()` it's already been resolved, as it happens the default behaviour of the messages api is to return the key as the value if it's not defined so these will still render the text but it would be clearer and more efficient to just use the resolved value as `LegendViewModel(heading).withCssClass("govuk-visually-hidden")`